recipes-bsp/uefi/files/0004-MdePkg-Base.h-handle-builtins-when-using-undef-flag.patch

```diff
@@ -0,0 +1,31 @@
+From 534f4b42e9821434448c1609f31253b1749062a0 Mon Sep 17 00:00:00 2001
+From: Jan Kircher <jan.kircher@leica-microsystems.com>
+Date: Thu, 5 Dec 2024 15:22:58 +0100
+Subject: [PATCH 1/1] MdePkg/Base.h: handle builtins when using -undef flag
+
+
+Upstream-Status: Pending
```

Comment:
Since this is a backport, the status should be 'Backport [https://github.com/tianocore/edk2/pull/5781]' so we have a reference to the upstream commit.